duckduckgo/Android
# Sync Engine: Account Flows and App Lifecycle Trigger

Task/Issue URL: https://app.asana.com/0/0/1204337450000851/f

### Description
Adds triggers to Sync after creating or login into an account

### Steps to test this PR
Check Asana task for testing scenarios

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204239356857194
